src/main/resources/content/tf/Azure-storage-account-logging-for-queues-is-disabled.json

```diff
@@ -0,0 +1,9 @@
+ {
+ "severity": "Medium",
+ "rule": "$.resource.*.azurerm_storage_account.*.*.*.queue_properties.*.logging.* size > 0 and ($.resource.*.azurerm_storage_account.*.*.*.queue_properties.*.logging.*.delete isFalse or $.resource.*.azurerm_storage_account.*.*.*.queue_properties.*.logging.*.read isFalse or $.resource.*.azurerm_storage_account.*.*.*.queue_properties.*.logging.*.write isFalse )",
```

Comment:
This will not work if there are multiple queue_account. You may need to use anyFalse instead of isFalse.